### PR TITLE
fix(adminmanual): Update grammar for calendar backend docs

### PR DESCRIPTION
**Pull request by jessebot**
### ☑️ Resolves

None, this just makes the docs more readable

### 🖼️ Screenshots

This is mostly fixing the word "back end" to be "[backend](https://en.wikipedia.org/wiki/Frontend_and_backend)" and adding commas and missing "s" where needed:
Screenshot 2023-07-02 at 18 15 58

### CODE REVIEW

**Review by miaulalala — APPROVED** [review]

File: admin_manual/groupware/calendar.rst [hunk 1]
```diff
@@ -115,13 +115,13 @@ Where the value is the number of seconds for the period. Setting the value to ``
...
 
-Once a back end app is installed the app typically allows admins or even users to define the resources, but this is subject of the specific implementation.
+Once a backend app is installed, the app typically allows admins, or even users, to define the resources, but this is subject of the specific implementation.
 
-Nextcloud periodically queries all registered back ends. Therefore new and updated resources and rooms will show with a delay.
+Nextcloud periodically queries all registered backends, therefore new/updated resources and rooms will show with a delay.
 
-Known back ends
+Known backends
 ~~~~~~~~~~~~~~~
```
Comment (ChristophWurst) [inline admin_manual/groupware/calendar.rst:125]:
```suggestion
~~~~~~~~~~~~~~
```

heading underline length has to match the text

**Pull request merged**